views/unacked.ecr

```diff
@@ -0,0 +1,35 @@
+<%- pagename = "Unacked" -%>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% render "head" %>
+    <script type="module" src="js/unacked.js"></script>
+  </head>
+  <body>
+    <% render "header" %>
+    <main class="unacked-main">
+      <section class="card">
+        <h3>
+          <a href="" id="queue-link"></a>
+          <small id="unacked-count"></small>
+        </h3>
+        <div class="table-wrapper">
+          <div id="table-error"></div>
+          <table id="table" class="table">
+            <thead>
+              <tr>
+                <th class="left">Message tag</th>
+                <th class="left">Consumer tag</th>
+                <th class="left">Channel</th>
+                <th class="left">Seconds unacked</th>
```

Comment:
Maybe we should decide on how to align different type of columns? My thinking is that integers should be right-aligned?